components/chart/pan-zoom.md

```diff
@@ -0,0 +1,388 @@
+        {
+            List<ChartSeriesData> data = new List<ChartSeriesData>();
+
+            for (int i = 1; i <= 10; i++)
+            {
+                var dataItem = new ChartSeriesData
+                    {
+                        ProductSales = i,
+                        Year = new DateTime(2000 + i, 3, i),
+                    };
+
+                data.Add(dataItem);
+            }
+
+            return data;
+        }
+    }
+}
+````
+
+### Specify Key and Lock Selected Axis for Panning
+
+* **Specify Key** - specify the key for performing panning through the `Key` parameter in the `ChartPannable` tag via the `ChartPannableKey` enum, which provides the following options:
+    * (default) `None` - no key is required
+    * `Ctrl`
+    * `Shift`
+    * `Alt`
+
+* **Lock Axis** - specifies an axis that should not be panned through the `Lock` parameter in the `ChartPannable` tag via the `ChartAxisLock` enum, which provides the following options:
+    * (default) `None` - none of the series are locked (as in, user can pan by either X and Y axis)
+    * `X` - X axis is locked, user can pan only by `Y` axis.
+    * `Y` - Y axis is locked, user can pan only by `X` axis.
+
+>caption Specified Key and Locked Axis.
+
+````CSHTML
+@* This code snippet showcases an example of pannable Chart with specified modifier key and locked Axis. *@
+Press CTRL + Click and Drag.
+<TelerikChart>
+    <ChartPannable Enabled="true"
+                   Lock="@ChartAxisLock.Y"
+                   Key="@ChartPannableKey.Ctrl">
+    </ChartPannable>
+
+    <ChartSeriesItems>
+        <ChartSeries Type="ChartSeriesType.Column"
+                     Name="Product 1"
+                     Data="@Data"
+                     Field="@nameof(ChartSeriesData.ProductSales)"
+                     CategoryField="@nameof(ChartSeriesData.Year)">
+        </ChartSeries>
+
+        <ChartCategoryAxes>
+            <ChartCategoryAxis Min="1" Max="5"></ChartCategoryAxis>
+        </ChartCategoryAxes>
+    </ChartSeriesItems>
+</TelerikChart>
+
+@code {
+    List<ChartSeriesData> Data { get; set; } = new List<ChartSeriesData>();
+
+    protected override Task OnInitializedAsync()
+    {
+        Data = ChartSeriesData.GenerateData();
+        return base.OnInitializedAsync();
+    }
+
+    public class ChartSeriesData
+    {
+        public int ProductSales { get; set; }
+        public DateTime Year { get; set; }
+
+        public static List<ChartSeriesData> GenerateData()
+        {
+            List<ChartSeriesData> data = new List<ChartSeriesData>();
+
+            for (int i = 1; i <= 10; i++)
+            {
+                var dataItem = new ChartSeriesData
+                    {
+                        ProductSales = i,
+                        Year = new DateTime(2000 + i, 3, i),
+                    };
+
+                data.Add(dataItem);
+            }
+
+            return data;
+        }
+    }
+}
+````
+
+## Configuring the Zoom Settings
```

Comment:
Consider making this a second article so that you have one for panning and one for zooming. As the structure of the zooming sections is similar to the structure of the panning sections, please apply the same suggestions here as well.